storybook/stories/Components.tsx

```diff
@@ -30,7 +32,17 @@ export default () =>
             : <div>loading...</div>
           )}
         </DAO.Data>
+        <MemberWrapper address="0xcb4e66eca663fdb61818d52a152601ca6afef74f">
```

Comment:
This is gross, but necessary... couldn't find a way around it...

Reply:
Re: "MemberWrapper"

Reply:
I don't think there is given how render methods work in React